code/modules/surgery/brain_surgery.dm

```diff
@@ -0,0 +1,41 @@
+/datum/surgery/brain_surgery
+	name = "brain surgery"
```

Comment:
rename to lobotomy

Reply:
if the subject is a plantperson it should be lobotany

Reply:
ahahahaha

Reply:
i'm torn on this, with brain surgery we'd get all the "it's not brain surgery" jokes

Reply:
Hmm